sdk/core/core-http/test/policies/tracingPolicyTests.ts

```diff
@@ -81,10 +89,8 @@ describe("tracingPolicy", function () {
   });
 
   it("will not create a span if spanOptions are missing", async () => {
-    const mockTracer = mockTracerFactory();
-    sinon.stub(TracerProxy, "getTracer").callsFake(() => {
```

Comment:
Since we're no longer stubbing the `TracerProxy.getTracer` method, the `afterEach` a few lines up can be updated as well.

Reply:
Good catch! 👍